can_tools/scrapers/official/WI/wi_state.py

```diff
@@ -33,23 +32,480 @@ def normalize(self, data):
         df["location"] = df["geoid"].astype(int)
 
         crename = {
-            "positive": CMU(category="cases", measurement="cumulative", unit="people"),
-            "deaths": CMU(category="deaths", measurement="cumulative", unit="people"),
-            "neg_new": CMU(
+            "positive": CMU(
+                category="pcr_tests_positive",
+                measurement="cumulative",
+                unit="unique_people",
+            ),
+            "negative": CMU(
                 category="pcr_tests_negative",
-                measurement="new",
+                measurement="cumulative",
                 unit="unique_people",
             ),
             "pos_new": CMU(
                 category="pcr_tests_positive",
                 measurement="new",
                 unit="unique_people",
             ),
+            "neg_new": CMU(
+                category="pcr_tests_negative",
+                measurement="new",
+                unit="unique_people",
+            ),
             "test_new": CMU(
                 category="pcr_tests_total",
                 measurement="new",
                 unit="unique_people",
             ),
+            "deaths": CMU(category="deaths", measurement="cumulative", unit="people"),
+            "dth_new": CMU(category="deaths", measurement="new", unit="people"),
+            "hosp_yes": CMU(
+                category="hospital_beds_in_use_covid",
+                measurement="cumulative",
+                unit="people",
+            ),
+        }
+        out = (
+            df.melt(id_vars=["location"], value_vars=crename.keys())
+            .assign(
+                dt=self._retrieve_dt("US/Central"), vintage=self._retrieve_vintage()
+            )
+            .dropna()
+        )
+        out.loc[:, "value"] = pd.to_numeric(out["value"])
+
+        # Extract category information and add other variable context
+        out = self.extract_CMU(out, crename)
+
+        cols_to_keep = [
+            "vintage",
+            "dt",
+            "location",
+            "category",
+            "measurement",
+            "unit",
+            "age",
+            "race",
+            "ethnicity",
+            "sex",
+            "value",
+        ]
+
+        return out.loc[:, cols_to_keep]
+
+
+class WisconsinState(ArcGIS, DatasetBase):
+    """
+    Fetch state-level covid data from Wisconsin's ARCGIS dashboard
+    Includes demographic breakdowns
+    """
+
+    ARCGIS_ID = ""  # not applicable on this server
+    has_location = True
+    location_type = "state"
+    state_fips = int(us.states.lookup("Wisconsin").fips)
+    source = "https://www.dhs.wisconsin.gov/covid-19/data.htm"
+
+    # # override query URL since Wisconsin has a self-hosted ArcGIS server with a different URL pattern
+    def arcgis_query_url(
```

Comment:
@MattSidor @sglyon 

To simplify this and ensure DRY, I created a new file called `common.py` and a `WisconsinArcGIS` class that inherits from `ArcGIS` and then has a single method which is `arcgis_query_url` then both `WisconsinState` and `WisconsinCounty` can inherit from this and not redefine this method. Thoughts?

Reply:
I'm ok with that. I don't think we necessarily need a common.py file, but I am not opposed to it

Reply:
See comments on separating the scrapers -- No need to pull this out.